hass-addon-sunsynk-multi/options.py

```diff
@@ -20,13 +20,18 @@ class InverterOptions:
     modbus_id: int = 0
     ha_prefix: str = ""
     serial_nr: str = ""
+    dongle_serial_number: str = ""
 
     @classmethod
     def factory(cls, opt: dict) -> InverterOptions:
         """Create a class from the options."""
         modbus_id = int(opt.pop("MODBUS_ID"))
+        dongle_serial_number = opt.pop("DONGLE_SERIAL_NUMBER")
```

Comment:
Since it's optional, test if it exists, or wrap in an try/catch with Indexerror

Reply:
I've made this change